src/plugins/intel_gpu/src/kernel_selector/cl_kernels/lora_ref.cl

```diff
@@ -0,0 +1,205 @@
+
+    const uint range_0_end = INPUT4_BATCH_NUM;
+    const uint range_1_end = range_0_end + INPUT7_BATCH_NUM;
+#if LORA_COUNT == 3
+    const uint range_2_end = range_1_end + INPUT10_BATCH_NUM;
+#endif
+
+    const uint proc_range_start[LORA_COUNT] = {
+        0, range_0_end
+#if LORA_COUNT == 3
+        , range_1_end
+#endif
+    };
+
+    const uint proc_range_end[LORA_COUNT] = {
+        range_0_end, range_1_end
+#if LORA_COUNT == 3
+        , range_2_end
+#endif
+    };
+
+    const __global STATE_TYPE* state_b[LORA_COUNT] = {
+        state_b_0, state_b_1
+#if LORA_COUNT == 3
+        , state_b_2
+#endif
+    };
+
+    __attribute__((opencl_unroll_hint(LORA_COUNT)))
+    for (uint i = 0; i < LORA_COUNT; ++i) {
+        uint tmp_offset = i * LORA_RANK;
+
+        uint processing_size = proc_range_end[i] - proc_range_start[i];
+        uint num_blocks = processing_size / LORA_RANK;
+        uint leftover = processing_size % LORA_RANK;
+
+        for (uint block = 0; block < num_blocks; ++block) {
+            uint yx_range = proc_range_start[i] + block * LORA_RANK + local_id;
+
+            ACCUMULATOR_TYPE final_acc = ACCUMULATOR_VAL_ZERO;
+
+            for (uint ki = 0; ki < LORA_RANK; ++ki) {
+                uint state_b_idx = (yx_range - proc_range_start[i]) * LORA_RANK + ki;
+
+                final_acc = mad(tmp_buf[tmp_offset + ki],
+                                TO_ACCUMULATOR_TYPE(state_b[i][state_b_idx]),
+                                final_acc);
+            }
+            uint output_idx = bf * INPUT0_SIZE_Y * INPUT0_SIZE_X + yx_range;
+            output[output_idx] = TO_OUTPUT_TYPE(final_acc) + main_input[output_idx];
+        }
+
+        if (local_id < leftover) {
+            uint yx_range = proc_range_start[i] + num_blocks * LORA_RANK + local_id;
+
+            ACCUMULATOR_TYPE final_acc = ACCUMULATOR_VAL_ZERO;
+
+            for (uint ki = 0; ki < LORA_RANK; ++ki) {
+                uint state_b_idx = (yx_range - proc_range_start[i]) * LORA_RANK + ki;
+
+                final_acc = mad(tmp_buf[tmp_offset + ki],
+                                TO_ACCUMULATOR_TYPE(state_b[i][state_b_idx]),
+                                final_acc);
+            }
+            uint output_idx = bf * INPUT0_SIZE_Y * INPUT0_SIZE_X + yx_range;
+            output[output_idx] = TO_OUTPUT_TYPE(final_acc) + main_input[output_idx];
+        }
+    }
+}
+
+#endif
+#endif
+
+#ifdef FUSED_OPS_KERNEL
+KERNEL(fused_ops)(OPTIONAL_SHAPE_INFO_ARG
+                  __global OUTPUT_TYPE* output
+#if HAS_FUSED_OPS_DECLS
+                , FUSED_OPS_DECLS
+#endif
+                    )
+{
+    const uint b = get_global_id(0);
+    const uint f = get_global_id(1);
+    const uint y = get_global_id(2) / OUTPUT_SIZE_X;
+    const uint x = get_global_id(2) % OUTPUT_SIZE_X;
+    const uint output_idx = OUTPUT_GET_INDEX(b, f, y, x);
+
+#if HAS_FUSED_OPS
+    FUSED_OPS;
+    output[output_idx] = FUSED_OPS_RESULT;
+#endif
+}
```

Comment:
Can we put fused_ops in the same kernel of lora? Thus post ops are computed in register level, and we reduce the extra overhead of more extra kernel.

Reply:
This transfer to new infra will be done in the next PR?

Reply:
Yes, this will be done in the next PR to protect the change from possible problems after the transfer